Fix/scripts search empty state (#658)
canonical/landscape-ui: .changeset/whole-colts-thank.md

---

> "landscape-ui": minor

Reviewer: I'd tag is a `patch` because it's rather a UX fix, not a new feature - minor inconsistency with how #644 (a similar "show a message when empty" change) was tagged `patch`.

---

> Improve scripts empty-state behavior by showing a search-specific no-results message when filters return zero matches, while keeping the default empty state for unfiltered views; adds test coverage for both paths.

Reviewer 2: This is more what the PR summary should be, not the changeset. 

The changeset message doesnt need to be so specific, we want it to be a very simple and clear description, since its for the users. They dont care about test coverage, for example.

```suggestion
Add a message for when scripts search returns no result 
```
